atomic_reactor/constants.py

```diff
@@ -74,6 +74,7 @@
 PLUGIN_FETCH_WORKER_METADATA_KEY = 'fetch_worker_metadata'
 PLUGIN_GROUP_MANIFESTS_KEY = 'group_manifests'
 PLUGIN_BUILD_ORCHESTRATE_KEY = 'orchestrate_build'
+PLUGIN_KOJI_PARENT_KEY = 'koji-parent'
```

Comment:
Should be an underscore, not a dash. Also, if adding this please also change the koji_parent plugin so it uses it, otherwise it gets defined in two places.